server/routes/reviewPlan/whoCompletedReviewController.test.ts

```diff
@@ -139,28 +139,5 @@ describe('whoCompletedReviewController', () => {
       expect(getPrisonerContext(req.session, prisonNumber).whoCompletedReviewForm).toBeUndefined()
       expect(getPrisonerContext(req.session, prisonNumber).reviewPlanDto).toEqual(reviewPlanDto)
     })
-
-    it(`should set backlinkUrl to Check Your Answers view given the previous page was Check Your Answers`, async () => {
```

Comment:
I dont think this test was testing anything over and above other tests in the class, and certainly nothing to do with the back link, so I've removed it